### ISSUE TITLE
Add check to ensure bump2version is installed

### ISSUE THREAD

**Issue by wslulciuc** [MarquezProject/marquez-python#45]
Let's add a simple check in [`release.sh`](https://github.com/MarquezProject/marquez-python/blob/master/release.sh) to ensure `bump2version` is installed before performing the release:

```
$ ./release.sh minor
++ git rev-parse --abbrev-ref HEAD
+ branch=master
+ [[ master != \m\a\s\t\e\r ]]
+ type=minor
+ '[' -z minor ']'
+ echo 'update type is: minor'
update type is: minor
++ python ./setup.py --version
+ version=0.3.2
+ echo 'Upgrading version from the current version of 0.3.2'
Upgrading version from the current version of 0.3.2
+ bumpversion --current-version 0.3.2 --commit --tag --tag-name '{new_version}' minor ./setup.py
./release.sh: line 42: bump2version: command not found
```

**Comment by ravikamaraj** [comment]
@wslulciuc , I can help on this one. I would reach out to you in case of any questions

**Comment by wslulciuc** [comment]
@ravikamaraj, thanks! 🥇

**Comment by ravikamaraj** [comment]
Hi @wslulciuc 
I have come up with below two different code snippets to check if bump2version is installed, can you review and let me know you thoughts, thanks!

**Option 1**
`bump2version -h > /dev/null 2>&1
	if [ $? -gt 0 ] ; then
		echo "Please install bump2version"
    exit 1;
	fi`

**Option 2**
`python -c "import bumpversion" > /dev/null 2>&1
	if [ $? -eq 1 ] ; then
		echo "Please install bump2version"
    exit 1;
	fi`

**Comment by wslulciuc** [comment]
Those options look great, @ravikamaraj! But what are your thoughts on a 3rd?

```bash
# Verify bump2version is installed
if [[ ! $(type -P bump2version) ]]; then
  echo "bump2version not installed! Please see https://github.com/c4urself/bump2version#installation"
  exit 1
fi
```

**Comment by ravikamaraj** [comment]
@wslulciuc, thanks!
The 3rd option looks great. I would test, and make the updates

**Comment by ravikamaraj** [comment]
it works great, thanks!

**Comment by wslulciuc** [comment]
Fixed: #63
